src/timings/Timings.php

```diff
@@ -299,4 +306,43 @@ public static function getEventHandlerTimings(string $event, string $handlerName
 
 		return self::$eventHandlers[$event][$handlerName];
 	}
+
+	public static function getAsyncTasksProgressUpdateTimings(AsyncTask $task) : TimingsHandler{
+		$taskClass = get_class($task);
+		if(!isset(self::$asyncTasksProgressUpdate[$taskClass])){
+			self::$asyncTasksProgressUpdate[$taskClass] = new TimingsHandler(
+				"AsyncTask Progress Update - " . $taskClass,
+				self::$schedulerAsync,
+				self::GROUP_BREAKDOWN
```

Comment:
This should probably be parameterized. We can't specify groups for this yet (we don't know which plugin it belongs to), but it'll likely be desirable in the future.